specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2023-07-01-preview/dataConnectorDefinitions.json

```diff
@@ -0,0 +1,758 @@
+            "description": "The connector is fully available to be used by customers."
+          },
+          {
+            "value": "FeatureFlag",
+            "description": "The connector is under a feature flag and not available for all customers."
+          },
+          {
+            "value": "Internal",
+            "description": "The connector is internal and should not be shown in DataConnectors blade."
+          }
+        ]
+      }
+    },
+    "ConnectivityCriterion": {
+      "description": "The criteria by which we determine whether the connector is connected or not.\r\nFor Example, use a KQL query to check if  the expected data type is flowing).",
+      "required": [
+        "type"
+      ],
+      "type": "object",
+      "properties": {
+        "type": {
+          "description": "Gets or sets the type of connectivity.",
+          "type": "string"
+        },
+        "value": {
+          "description": "Gets or sets the queries for checking connectivity.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "ConnectorDataType": {
+      "description": "The data type which is created by the connector,\r\nincluding a query indicated when was the last time that data type was received in the workspace.",
+      "required": [
+        "lastDataReceivedQuery",
+        "name"
+      ],
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Gets or sets the name of the data type to show in the graph.",
+          "type": "string"
+        },
+        "lastDataReceivedQuery": {
+          "description": "Gets or sets the query to indicate when relevant data was last received in the workspace.",
+          "type": "string"
+        }
+      }
+    },
+    "ConnectorDefinitionBase": {
+      "description": "An Azure resource, which encapsulate the entire info requires to display a data connector page in Azure portal,\r\nand the info required to define data connections.",
+      "type": "object",
+      "properties": {
+        "createdTimeUtc": {
+          "format": "date-time",
+          "description": "Gets or sets the connector definition created date in UTC format.",
+          "type": "string"
+        },
+        "lastModifiedUtc": {
+          "format": "date-time",
+          "description": "Gets or sets the connector definition last modified date in UTC format.",
+          "type": "string"
+        }
+      }
+    },
+    "ConnectorDefinitionBaseArmCollectionWrapper": {
+      "type": "object",
+      "description": "Encapsulate the data connector definition object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ConnectorDefinitionBaseArmObjectWrapper"
+          }
+        },
+        "nextLink": {
+          "type": "string"
+        }
+      }
+    },
+    "ConnectorDefinitionBaseArmObjectWrapper": {
```

Comment:
It seems that `ConnectorDefinitionBaseArmObjectWrapper` and `ConnectorDefinitionBaseArmObjectWrapperWithConverter` use same definition but slightly different description?

It's not necessary to do it, please use same definition.

Reply:
in our code we have 2 object with the same properties, we need to have different object because we are using different converter and this cant be change.

we have a plan to generate the swagger doc from the code,  this file was partially generated from the code, and that's why there are different names from the same object.

WDYT?